### PR TITLE
Update SDK version + ActivityFull type

### PR DESCRIPTION
**Pull request by nicoback2**
### Description
Needed for library changes; wanted to add to main instead of the feature branch.

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype

File: packages/common/src/services/audius-api-client/ResponseAdapter.ts [hunk 1]
```diff
@@ -390,13 +391,17 @@ export const makePlaylist = (
...
+  activity: APIActivity | APIActivityV2
 ): UserTrackMetadata | UserCollectionMetadata | undefined => {
-  switch (activity.item_type) {
+  const itemType =
+    'item_type' in activity ? activity.item_type : activity.itemType
+  switch (itemType) {
     case 'track':
-      return makeTrack(activity.item)
+      // Cast to APITrack because TS is not correctly inferring the type
+      return makeTrack(activity.item as APITrack)
```
Comment (dylanjeffers) [inline packages/common/src/services/audius-api-client/ResponseAdapter.ts:401]: interesting, just curious what .item is before the cast?

File: packages/common/src/services/sdk/storageNodeSelector.ts [hunk 2]
```diff
@@ -8,6 +8,8 @@ import { Env } from '../env'
 import { getBootstrapNodes } from './bootstrapNodes'
 import { DiscoveryNodeSelectorService } from './discovery-node-selector'
 
+const logger = new Logger()
```
Comment (dylanjeffers) [inline packages/common/src/services/sdk/storageNodeSelector.ts:11]: kewl

**Review by dylanjeffers — APPROVED** [review]
great stuff, just looks ilke some type issues

File: packages/common/src/services/sdk/storageNodeSelector.ts (hunk 2 above)
Comment (rickyrombo) [reply packages/common/src/services/sdk/storageNodeSelector.ts:11]: Shouldn't be necessary in latest SDK! :)

File: packages/common/src/services/audius-api-client/ResponseAdapter.ts (hunk 1 above)
Comment (sliptype) [reply packages/common/src/services/audius-api-client/ResponseAdapter.ts:401]: I wonder if you could use a TS type guard to be explicit about `APIActivity` vs `APIActivityV2`, that may help with the inference here

here's an example https://github.com/AudiusProject/audius-protocol/blob/main/libs/src/sdk/types/File.ts#L31

**Review by sliptype — APPROVED** [review]

**Comment by audius-infra** [comment]
It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype

**Comment by audius-infra** [comment]
It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--update-sdk-activity

**Pull request merged**